images/kubevirt-infra-prow-arm64/builder/nodejs.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/WORKSPACE b/WORKSPACE
+index d500c17519..fc11d57dde 100644
+--- a/WORKSPACE
++++ b/WORKSPACE
+@@ -62,7 +62,12 @@ load("@io_bazel_rules_k8s//k8s:k8s.bzl", "k8s_repositories")
+
+ k8s_repositories()
+
+-load("@build_bazel_rules_nodejs//:defs.bzl", "yarn_install")
++load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories", "yarn_install")
```

Comment:
The changes here look minimal, any chance that we can just push them right now into test-infra? They may not care right now about multi-arch builds, but they should not mind merging a fix. What exactly are you fixing?

Reply:
> The changes here look minimal, any chance that we can just push them right now into test-infra? They may not care right now about multi-arch builds, but they should not mind merging a fix. What exactly are you fixing?

The bazel rule "@build_bazel_rules_nodejs//:defs.bzl" only support x86 arch. Add arm64 arch nodejs source code and binary, thus "bazel build" in k8s.io/test-infra project can be used in Arm64 platform. 
May need to send PR to bazel project to fix it. I am not quite familar about it, need to do more research.